src/main/java/StringCalculatorStep1.java

```diff
@@ -5,7 +5,8 @@ public static final int Add(final String numbers) {
         int returnValue = 0;
         String[] numbersArray = numbers.split(",");
         if (numbersArray.length > 2) {
-            throw new RuntimeException("Until 2 numbers seperated by commas is allowed");
+            throw new RuntimeException("Up to 2 numbers separated by comma (,) are allowed");
+
```

Comment:
The exception message is improved, but consider externalizing it for better maintainability.